core/data-plane/data-plane-core/src/test/java/org/eclipse/edc/connector/dataplane/framework/manager/DataPlaneManagerImplTest.java

```diff
@@ -393,10 +393,10 @@ void completed_shouldNotifyResultToControlPlane() {
     }
 
     @Test
-    void completed_shouldNotTransitionToNotified() {
+    void completed_whenRetryableError_shouldTransitionToCompleted() {
```

Comment:
please adapt the test naming format as convention:
`shouldDoSomething_whenSomethingElseHappens`
for all the tests here